fe/fe-core/src/main/java/com/starrocks/backup/RestoreJob.java

```diff
@@ -843,7 +843,7 @@ private Partition resetPartitionForRestore(OlapTable localTbl, OlapTable remoteT
                 List<Long> beIds =
                         GlobalStateMgr.getCurrentSystemInfo().seqChooseBackendIds(restoreReplicationNum, true,
                                 true);
-                if (beIds == null) {
+                if (beIds.isEmpty()) {
```

Comment:
`CollectionUtils.isEmpty` is more safe? To check null or empty?

Reply:
Currently, it won't return null, but anyway using `CollectionUtils.isEmpty` looks like a better practice. I will refine it.